railgun/test/integration/ingress_test.go

```diff
@@ -81,6 +82,63 @@ func TestMinimalIngress(t *testing.T) {
 		return false
 	}, ingressWait, waitTick)
 
+	t.Logf("removing the ingress.class annotation \"%s\" from ingress %s", ingressClass, ingress.Name)
```

Comment:
```suggestion
	t.Logf("removing the ingress.class annotation %q from ingress %s", ingressClass, ingress.Name)
```

Reply:
this also applies to other instances of this pattern

Reply:
Good call, I missed this when re-reading my own code :+1: 

resolved by d11036a8b66aa1dec2c3654b09892595fa6cf657